### PR TITLE
docs(agw): Updated quick start guide to add ipv6 range in /etc/vbox/networks.conf

### LINKED ISSUE #10328: bug(agw): invalid IP address in Vagrantfile
### Your Environment

- **OS Version:** MacOS Monterey 12.0.1
- **Affected Component:** Access Gateway
- **Affected Subcomponent:** Vagrant
- **Deployment Environment:** Vagrant (AGW)

### Describe the Issue

Provisioning the AGW VM per the Quick Start leads to error "IP address configured for the host-only network is not within the allowed ranges"

**To Reproduce**

1. Run through Quick Start to "Terminal Tab 1: Provision the AGW VM" section
2. `cd lte/gateway`
3. `vagrant up magma`

<pre>The IP address configured for the host-only network is not within the
allowed ranges. Please update the address used to be within the allowed
ranges and run the command again.

  Address: 192.168.129.1
  Ranges: 192.168.56.0/21</pre>

**Expected behavior**
VirtualBox should boot

**Screenshots**

image

**Additional context**

The IP is set in `magma/lte/gateway/Vagrantfile`

### PR DESCRIPTION
**Pull request by pruthvihebbani**
Signed-off-by: Pruthvi Hebbani <pruthvi.hebbani@radisys.com>

docs(agw): Updated quick start guide to add ipv6 range in /etc/vbox/networks.conf

## Summary

Updated quick start guide to add ipv6 range in /etc/vbox/networks.conf

## Test Plan
Verified that magma_dev, magma_test and trf_server VMs are successfully coming up in my local setup.
image
image
image

### CODE REVIEW

**Review by Neudrino — CHANGES_REQUESTED** [review]
- This change will also be needed in the current master version of the documentation.
- No test plan was given.
  - Expected result should be given.
  - Test plan should be executed and actual result should be given.

**Review by ssanadhya — APPROVED** [review]
Lgtm

File: docs/docusaurus/versioned_docs/version-1.7.0/basics/quick_start_guide.md [hunk 1]
```diff
@@ -36,11 +36,12 @@ any production hardware on hand to test an end-to-end setup.
 We'll be setting up the LTE AGW VM in this tab.
```
Comment (nstng) [inline docs/docusaurus/versioned_docs/version-1.7.0/basics/quick_start_guide.md]: My understanding is that this should not be added to the 1.7 documentation. 1.7 does not have this change. I assume the change will not be back ported to 1.7, right?

**Review by Neudrino — APPROVED** [review]
Did you know that there is a ["Re-request review" feature](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review), which should be used to bring PRs to reviewers attention again?
Otherwise, they will not know, that the PR is ready for another review.

grafik

**Comment by pruthvihebbani** [comment]
> Did you know that there is a ["Re-request review" feature](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review), which should be used to bring PRs to reviewers attention again? Otherwise, they will not know, that the PR is ready for another review.
> 
> grafik

I was not aware of this. Thanks for the info @Neudrino

**Pull request merged**